articles/client-platforms/ember2js.md

```diff
@@ -28,10 +28,10 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../_includes/_package', {
-  pkgRepo: 'auth0-ember-simple-auth',
+  pkgRepo: 'auth0-emberjs2-spa',
```

Comment:
Is this the one we're looking for? https://github.com/auth0-samples/auth0-emberjs2-sample